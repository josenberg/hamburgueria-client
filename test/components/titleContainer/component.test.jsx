import React from 'react';
import renderer from 'react-test-renderer';

import TitleContainer from '../../../app/components/titleContainer';
// Redux store


test('Header should render without issues', () => {
  const component = renderer.create(<TitleContainer />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
