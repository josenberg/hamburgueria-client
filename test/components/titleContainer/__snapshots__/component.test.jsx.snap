// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render without issues 1`] = `
<div
  className="grid"
  style={
    Object {
      "backgroundColor": "#7F171F",
      "border": "5px solid #7F5417",
      "colGap": "0px",
      "color": "white",
      "display": "grid",
      "fontFamily": "'Lalezar', cursive",
      "gridTemplateColumns": "1fr auto",
      "gridTemplateRows": "1",
      "margin": "0 auto",
      "rowGap": "0px",
      "width": "400px",
    }
  }
>
  <div
    className="grid-cell"
    style={
      Object {
        "display": "grid",
        "gridColumn": "1",
        "gridRow": "auto",
      }
    }
  >
    <h1
      style={
        Object {
          "marginBottom": "0px",
          "textAlign": "center",
        }
      }
    >
      
    </h1>
    <a
      href=""
      onClick={[Function]}
      style={
        Object {
          "color": "white",
          "paddingRight": "5px",
          "textAlign": "right",
        }
      }
    >
      
    </a>
  </div>
</div>
`;
